### PR TITLE
Playlist 

### PR DESCRIPTION
**Pull request by tanishk15sharma**
**Playlist Management**

_Style_

- Playlists page
- Playlist box
- Playlist video container
- Add Playlist `input` and `button`
- Playlist Modal 

_Functionility_ 

- `GET` playlists 
- `POST` playlist
- `DELETE` playlist
- Add video to playlist
- Remove video from playlist

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import "./App.css";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
-import { Videos, WatchLater, Like, VideoPlayer } from "./pages";
+import { Videos, WatchLater, Like, VideoPlayer, Playlists } from "./pages";
```
Comment (Mohitkumar33) [inline src/App.js:3]: It is nice that you are importing all from a single file

**Review by Mohitkumar33 — APPROVED** [review]
Nicely arranged all the files. Code is looking clean.

**Review by agrawalshanu708 — APPROVED** [review]
Code structure is good,
Code is clean
You can merge

File: src/components/playlists/PlaylistListing.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
+.playlists-wrapper {
+  border: 1px solid red;
+  padding: 2rem 4rem;
+}
+.playlist-box {
+  background-color: var(--background-secondary-color);
+  padding: 1rem 2rem;
+  border-radius: 10px;
+  cursor: pointer;
+  /* border: 1px solid white; */
```
Comment (deepak29-git) [inline src/components/playlists/PlaylistListing.css:10]: remove commented code

File: src/App.js (hunk 1 above)
Comment (tanishk15sharma) [reply src/App.js:3]: Yes , doing this so that my `aap.js` will look clean and defined.

File: src/utilities/playlist-utils.jsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    });
+    dispatch({ type: "DELETE_FROM_PLAYLISTS", payload: data.playlists });
+  } catch (err) {
+    console.log(err);
+  }
+};
+// playlist
+
+// const setPlaylist = async (id) => {
+//   try {
```
Comment (deepak29-git) [inline src/utilities/playlist-utils.jsx:47]: remove unused code

File: src/contexts/user-videos.jsx [hunk 4]
```diff
@@ -2,29 +2,54 @@ import { createContext, useContext, useEffect, useReducer } from "react";
 import { getLikesArr } from "../utilities/likes-utils";
 import { getWatchLaterArr } from "../utilities/watchlater-utils";
 import { getAllVideos } from "../utilities/allvideos-utils";
+import { getPlaylists } from "../utilities/playlist-utils.jsx";
 const UserVideosContext = createContext();
 
 const userVideosReducer = (state, action) => {
```
Comment (deepak29-git) [inline src/contexts/user-videos.jsx:8]: you can abstract this to another file so it look more cleaner and easy to debug (just for suggestion)

**Review by deepak29-git — APPROVED** [review]

**Pull request merged**